ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -146,6 +146,17 @@ tests:
     test:
     - ref: windows-e2e-operator-test-byoh
     workflow: windows-e2e-operator-test-platform-none-vsphere
+- as: wicd-unit-vsphere
+  steps:
+    cluster_profile: vsphere-platform-none
```

Comment:
I suggest using `cluster_profile: vsphere`.

Reply:
any reason for choosing platform-none?

Reply:
I don't believe its possible to use `cluster_profile: vsphere`

Reply:
> any reason for choosing platform-none?

Yes because it allows for us to spin up resources and have network access to them without creating an OpenShift cluster

Reply:
AFAIK, the `cluster_profile` just provides the `LEASED_RESOURCE`.  The proposed VMs can be created in any ci-segment, I don't see any constraints.

In addition, using `vsphere` as cluster_profile will alleviate the leasing resources for `platform-none` jobs, where there are only 4 ci-segments.

Reply:
> Is this job able to run in either?

I don't see why not, I might be missing something here though

>  How many concurrent instances of this job do you think will be running?

Hopefully not too many :)
This should be a quicker job, but it will run against all WMCO PRs